charts/pravega/values.yaml

```diff
@@ -109,4 +109,58 @@ storage:
     className: standard
 
 options:
-  # bookkeeper.ack.quorum.size: "3"
+  ## If you add more Bookies to the cluster and you want to exploit them for IO, then you need increase this number. The Bookkeeper ensemble defines
```

Comment:
Please, remove some of these explanations, I don't think they are needed.

Reply:
done